### PR TITLE
M4 stretch tech

### LINKED ISSUE #322: Create Solo Stretch Tech Project
Create a stretch tech project for M4 students that kicks off in week 5 and is a "ten week project" to get them working on something specific after graduation.

Update the Final Presentation guidelines to have students outline their plans. They can show their repo, project board, user stories, calendar, etc.

There are lots of notes about these changes [here](https://www.notion.so/turingschool/2410-W1-Mod-4-Planning-10e0ef35707881e8be31d4ae36748328) and full discussion can be found in [this recording](https://turingschool.slack.com/files/UJ2H4M63A/F07R6FSSK35/video1141639882.mp4) (messaged 10/9 in #software-engineering-program)

### PR DESCRIPTION
**Pull request by kaylagordon**
# Curriculum PR Template

### Does this PR close any issues?
This PR closes #322 

### Description
New stretch tech project for M4 students to kick off on Monday of week 5.

You will want to look at the project spec AND this [deliverables google sheet](https://docs.google.com/spreadsheets/d/1dkhqQctc_6nylYyQII22zLU0He_P1k_xfaJOFngV7dM/edit?gid=0#gid=0). 

### What questions do you have/what do you want feedback on? (optional)
- All my questions are listed as comments.

### CODE REVIEW

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:48]: Flagging this comment for BE eyes

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:28]: Do we want to add Auth? CI/CD? Where would those fit into the "menus" below?

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:116]: Do we need a rubric? Or is this sufficient?

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:57]: The only thing that I think an instructor needs to give in-the-moment feedback on in this list is the Project Proposals on Day 2. I think the rest of the time the instructor can keep an eye on the deliverables sheet and follow up as needed. If they have more capacity, they can dig into the design docs and give feedback.

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:5]: I put everything on one page, but feel free to split it up if you think that's better.

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [inline module4/projects/solo_stretch_tech/index.md:60]: Do the deliverables seem like over-management or supportive to get them to focus on the right things? I want them to do minimal coding and focus on planning. I figured having super outlined deliverables could help with that?

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:28]: I was actually thinking CI/CD should be required, not stretch, and due with the day 7 repo deliverables. Auth could go either way. I think I'd say OAuth is stretch, regular auth feels iffier to me.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:48]: These all look like great stretches to me

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:60]: I think this sounds like a good place to start. Yes they did a lot of this in consultancy, but it was with a group. I think the handholding will be helpful in addition to keeping them accountable.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:72]: Propose changing this to Day 5. The first week of Stretch coincides with Full Interview simulations. I'm not going to have time to prep interview docs and review project proposals on the same day which would have to happen if we don't want them to move on to these Day 4 (or 5) deliverables before getting instructor approval.

Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:72]: I think the rest of the schedule can remain unchanged. They'll still have the whole weekend to work on getting repos set up and issues written for the Day 7 deliverable.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:83]:
```suggestion
- Repo(s): Please create your repo(s) and link them in the google sheet. You don't need to have any code in there yet.
- CI/CD: Set up continuous integration on both repos. We recommend using Github actions or CircleCI for this
- Project Board: You will link your project board, which should include:
```

Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:83]: I also kind of want to add that they should deploy this via Vercel and Render (or other free service of their choosing) since I think both of those are still free. That would make sense to be due with the Day 7 deliverables as well to encourage Deploy Early, Deploy Often

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:95]: I might just be way too obtuse, but I read this as students both start the PR and submit/merge all on Day 9. I'd love a normal person's take on if the read it the same way or if we expect students will spend both days 8 and 9 on this. If we think there will be confusion, it might be worth clarifying that Day 8 should be the start of coding.

Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:95]: And maybe also a reminder in this Day 9 dropdown that if they cannot finish the issue between days 8-9, the issue is too large and should be broken out into smaller tasks.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:116]: I'm leaning yes. It's listed as a graduation requirement that a student passes the stretch tech project, so it should probably be pretty clear what "passing" looks like. Right now this reads as being focused only on the quality of the presentation.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:105]:
```suggestion
## Presentation and Evaluation Details
```

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:137]: I think we can remove these two topics because this is pretty heavily covered in the Case Study project which is due in Week 5. I like the idea of the presentation focusing only on the Stretch project and how they can use the project and learnings on their job hunt (versus higher level job hunt questions).

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:139]:
```suggestion
  - Show us your calendar for the first 2 weeks post-graduation. This should include time for working on your Stretch Tech project AND job hunt activities.
```

**Review by epintozzi — COMMENTED** [review]
Love this.

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [reply module4/projects/solo_stretch_tech/index.md:116]: Rubric category ideas:
* Presentation
* Work Ethic (meets due dates)
* Preparation (completes deliverables thoroughly/as we intended)

File: module4/projects/solo_stretch_tech/index.md
Comment (epintozzi) [inline module4/projects/solo_stretch_tech/index.md:108]:
```suggestion
Your presentation will highlight your progress on this project and your plans for your job search moving forward. It’s an opportunity to reflect on your experiences and demonstrate how you’ve grown and learned throughout the process.
```

File: module4/projects/solo_stretch_tech/index.md
Comment (kaylagordon) [reply module4/projects/solo_stretch_tech/index.md:96]: I can add a note to the end of Day 7 and say "It's time to start coding!"

**Review by epintozzi — APPROVED** [review]

**Pull request merged**
